src/components/Signin/Signin.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Form, Modal, Popup, Icon } from 'semantic-ui-react';
+
+import SignInButton from '../Shared/SignInButton'
+import TabBar from '../TabBar/TabBar';
+
+const ModalSignIn = (
+  {
+    submittedName,
+    submittedEmail,
+    handleChange,
+    handleSubmit,
+    signInSubmit,
+    validations,
+    upClose,
+    inModalOpen,
+    inCloseConfig,
+    signBothClose,
+    usernameSignIn='',
+    passwordSignIn = ''
+  }
+) => (
+    <Modal
+      trigger={<SignInButton closeConfig={inCloseConfig} />}
+      size='tiny'
+      open={inModalOpen}
+      onClose={signBothClose}
+      closeOnDimmerClick={true}
+    >
+      <Modal.Header><TabBar tabMessage={'S i g n_I n . e x e'} closeModal={signBothClose} /></Modal.Header>
```

Comment:
Only use curly braces when trying to write JS code. So instead of `{'S i g n_I n . e x e'}`, just write `''S i g n_I n . e x e'}'`